python/lsst/afw/math/spatialCell.cc

```diff
@@ -89,105 +222,30 @@ void declareTestClasses(py::module &mod){
-                       "label"_a,
-                       "bbox"_a=lsst::afw::geom::Box2I(),
-                       "candidateList"_a=CandidateList());
-    /* SpatialCell Members */
-    clsSpatialCell.def("getLabel", &SpatialCell::getLabel);
-    clsSpatialCell.def("begin", (SpatialCellCandidateIterator (SpatialCell::*)()) &SpatialCell::begin);
-    clsSpatialCell.def("begin", (SpatialCellCandidateIterator (SpatialCell::*)(bool)) &SpatialCell::begin);
-    clsSpatialCell.def("end", (SpatialCellCandidateIterator (SpatialCell::*)()) &SpatialCell::end);
-    clsSpatialCell.def("end", (SpatialCellCandidateIterator (SpatialCell::*)(bool)) &SpatialCell::end);
-    clsSpatialCell.def("insertCandidate", &SpatialCell::insertCandidate);
-    clsSpatialCell.def("size", &SpatialCell::size);
-    clsSpatialCell.def("setIgnoreBad", &SpatialCell::setIgnoreBad);
-    clsSpatialCell.def("getCandidateById",
-                        &SpatialCell::getCandidateById,
-                        "id"_a,
-                        "noThrow"_a=false);
-    clsSpatialCell.def("sortCandidates", &SpatialCell::sortCandidates);
-    clsSpatialCell.def("empty", &SpatialCell::empty);
-    clsSpatialCell.def("getBBox", &SpatialCell::getBBox);
-    
-    /* SpatialCellSet */
-    py::class_<SpatialCellSet, std::shared_ptr<SpatialCellSet>>
-        clsSpatialCellSet(mod, "SpatialCellSet");
-    clsSpatialCellSet.def(py::init<lsst::afw::geom::Box2I const&, int, int>(),
-                          "region"_a, "xSize"_a, "ySize"_a=0);
-    clsSpatialCellSet.def("getCellList", &SpatialCellSet::getCellList);
-    clsSpatialCellSet.def("insertCandidate", &SpatialCellSet::insertCandidate);
-    clsSpatialCellSet.def("visitCandidates", 
-                          (void (SpatialCellSet::*)(CandidateVisitor *, int const, bool const))
-                              &SpatialCellSet::visitCandidates,
-                          "visitor"_a, "nMaxPerCell"_a=-1, "ignoreExceptions"_a=false);
-    clsSpatialCellSet.def("visitCandidates",
-                          (void (SpatialCellSet::*)(CandidateVisitor *, int const, bool const) const)
-                              &SpatialCellSet::visitCandidates,
-                          "visitor"_a, "nMaxPerCell"_a=-1, "ignoreExceptions"_a=false);
-    clsSpatialCellSet.def("getCandidateById", &SpatialCellSet::getCandidateById, "id"_a, "noThrow"_a=false);
-    clsSpatialCellSet.def("sortCandidates", &SpatialCellSet::sortCandidates);
-    
-    /* CandidateVisitor */
-    py::class_<CandidateVisitor, std::shared_ptr<CandidateVisitor>>
-        clsCandidateVisitor(mod, "CandidateVisitor");
-    
-    
-    
-    /* Module level */
-
-    /* Member types and enums */
-
-    /* Constructors */
-
-    /* Operators */
-
-    /* Members */
+    wrapSpatialCellCandidate(mod);
+    wrapSpatialCellCandidateIterator(mod);
+    wrapSpatialCell(mod);
+    wrapSpatialCellSet(mod);
+    wrapCandidateVisitor(mod);
+    wrapSpatialCellImageCandidate(mod);
```

Comment:
Because there were too darn many classes. I felt it made the code very hard to read, partly due to clutter and partly due to the long class names being repeated many times, making for long lines.